apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -56,17 +56,30 @@ const useSections = section => {
         ]
   );
 
-  const updateSection = (sectionIdx, keyToUpdate, val) => {
-    const newSections = sections.map((section, idx) => {
-      if (idx === sectionIdx) {
-        return {
-          ...section,
-          [keyToUpdate]: val,
-        };
-      } else {
-        return section;
-      }
-    });
+  const updateSection = (sectionIdx, keyToUpdate, val, updateList = null) => {
+    let newSections;
+    if (!keyToUpdate && !val) {
```

Comment:
I think it would be cleaner and easier to read if we had two functions `updateSection` and `updateSectionWithList`

Reply:
or `batchUpdateSection`